src/quotamodel.c

```diff
@@ -540,10 +532,11 @@ DiskQuotaShmemSize(void)
 	if (IS_QUERY_DISPATCHER())
 	{
 		size = add_size(size, diskquota_launcher_shmem_size());
-		size = add_size(size, sizeof(pg_atomic_uint32));
 		size = add_size(size, diskquota_worker_shmem_size() * diskquota_max_monitored_databases);
 		size = add_size(size, hash_estimate_size(MAX_QUOTA_MAP_ENTRIES, sizeof(QuotaInfoEntry)) *
 		                              diskquota_max_monitored_databases);
+		size = add_size(size,
```

Comment:
Why is this line not in the diskquota_worker_shmem_size function?

Reply:
because, the size of the quota_info_map hash table is added here.

Reply:
It would be more logical to move this size to that function.